nimsuggest/tester.nim

```diff
@@ -13,16 +15,16 @@ type
     disabled: bool
 
 const
-  curDir = when defined(windows): "" else: ""
   DummyEof = "!EOF!"
-
-template tpath(): untyped = getAppDir() / "tests"
+  tpath = "nimsuggest/tests"
+  # we could also use `stdtest/specialpaths`
 
 import std/compilesettings
 
 proc parseTest(filename: string; epcMode=false): Test =
   const cursorMarker = "#[!]#"
-  let nimsug = curDir & addFileExt("nimsuggest", ExeExt)
```

Comment:
previous code was picking up the nimsuggest in PATH instead of the one in bin/nimsuggest that was just built in `runCI`; for CI it doesn't matter since there's only one, but it matters when testing and user has installed nim prior to running this. If needed i can add an option to pass nimsuggest binary via `--nimsuggest:exe`

new behavior is more consistent with how we test other tools